QuickLook.Plugin.FolderViewer.csproj

```diff
@@ -103,4 +103,7 @@
     </Page>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>rem powershell -file "$(SolutionDir)Scripts\pack-zip.ps1"</PostBuildEvent>

```

Comment:
What does this do? Executing `pack-zip.ps1` makes sense. Why is it prefixed with `rem` to disable it?

Reply:
It doesn't work. You can pull and try.
Hope can run pack-zip.ps1 after building, but it's not urgent. So comment the script.
The tag should be 1.2 rather than v1.2 in GitVersion.cs generate by QuickLook.Common.

Reply:
Yup, got that when I set up appveyor yesterday. I corrected the tag to 1.2. Should work fine now.